ledger/storage/drop/drop.go

```diff
@@ -27,6 +27,7 @@ import (
 //go:generate minimock -i github.com/insolar/insolar/ledger/storage/drop.Modifier -o ./ -s _mock.go
 type Modifier interface {
 	Set(ctx context.Context, jetID core.JetID, drop jet.Drop) error
+	Delete(pulse core.PulseNumber)
```

Comment:
Let's move it to a separate interface.
DB implementation should not have this method. This way it's impossible to misuse the component instead of runtime error.